packages/generator-joplin/generators/app/templates/webpack.config.js

```diff
@@ -30,6 +30,7 @@ const userConfig = Object.assign({}, {
 const manifestPath = `${srcDir}/manifest.json`;
 const packageJsonPath = `${rootDir}/package.json`;
 const allPossibleCategories = ['appearance', 'developer tools', 'productivity', 'themes', 'integrations', 'viewer', 'search', 'tags', 'editor', 'files', 'personal knowledge management'];
+const allPossibleScreenshotsType = ['jpg', 'png', 'gif', 'webp', 'jepg'];
```

Comment:
jepg

Reply:
Yes, I've run it with my plugin emoji picker. It packs all files under src since the original webpack script does so and I don't need to do anything to pack the screenshots. 
I've tested the cases that 
- the src is not specified
- the screenshot specified in src does not exist
- the extension is not in the list
- the screenshot is bigger than 1mb
and they work as expected.